src/test/tv/codely/scala_http_api/module/video/application/create/VideoCreatorTest.scala

```diff
@@ -0,0 +1,18 @@
+package tv.codely.scala_http_api.module.video.application.create
+
+import tv.codely.scala_http_api.module.video.VideoUnitTestCase
+import tv.codely.scala_http_api.module.video.domain.VideoStub
+
+final class VideoCreatorTest extends VideoUnitTestCase {
+  private val creator = new VideoCreator(repository)
+
+  "Videos Creator" should {
+    "save a video" in {
+      val video = VideoStub.random
+
+      shouldSaveVideo(video)
```

Comment:
it feels weird having a call such as this without any reference to the repository. You know what this method does. But if you end up in this test for the very first time, it wouldn't be explicit enough.

What I'm proposing here is to assume as a convention that these mocks helper methods always explicit the name of the mock they're using. Something like: `%concept%Should%method%(%value%)`. For this very own case, it would be: `repositoryShouldSave(video)` or even `videoRepositoryShouldSave(video)`.

What do you think?

Reply:
`shouldSaveInVideoRepository`?

Reply:
Given we only have 1 repo/module -> `shouldSaveInRepository`